client/assets/stylesheets/_main.scss

```diff
@@ -216,6 +216,7 @@ kbd,
 tt,
 var {
 	/* stylelint-disable-next-line scales/font-size */
+	/* stylelint-disable-next-line declaration-property-unit-allowed-list */
```

Comment:
curious if we can get rid of one of the rules here or move them into the same line

Reply:
Removed